cfme/configure/configuration/__init__.py

```diff
@@ -423,8 +428,10 @@ class ServerLogDepot(Pretty):
 
     """
     elements = Region(
-        locators={},
-        infoblock_type="form"
+        locators={
+            "last_message": InfoBlock.create("Basic Info", "Last Message"),
+            "last_log_collection": InfoBlock.create("Basic Info", "Last Log Collection"),
+        },
```

Comment:
The only thing I don't like about this is that I now have to define each key/value I want to target first in locators, leading to double the lines for the same effect.